src/test/scala/mesosphere/marathon/integration/ResidentTaskIntegrationTest.scala

```diff
@@ -0,0 +1,105 @@
+package mesosphere.marathon.integration
+
+import mesosphere.marathon.api.v2.json.AppUpdate
+import mesosphere.marathon.integration.setup.MarathonFacade._
+import mesosphere.marathon.integration.setup.{ SingleMarathonIntegrationTest, IntegrationFunSuite }
+import mesosphere.marathon.state._
+import org.apache.mesos.{ Protos => Mesos }
+import org.scalatest.{ GivenWhenThen, BeforeAndAfter, Matchers }
+import org.slf4j.LoggerFactory
+import scala.collection.immutable.Seq
+
+class ResidentTaskIntegrationTest
+    extends IntegrationFunSuite
+    with SingleMarathonIntegrationTest
+    with Matchers
+    with BeforeAndAfter
+    with GivenWhenThen {
+
+  private[this] val log = LoggerFactory.getLogger(getClass)
+
+  //clean up state before running the test case
+  before(cleanUp())
+
+  // FIXME(ME) enable the test when the base functionality is implemented
+  ignore("persistent volume will be re-attached and keep state") {
+    import StatusUpdateEvent._
+
+    val f = new Fixture
+    val appId = testBasePath / "app"
+    val containerPath = "/mnt/persistent-volume"
+
+    Given("A task that writes into a persistent volume")
+    create(f.appWithPersistentVolume(
+      appId,
+      containerPath,
+      cmd = s"""echo "data" > $containerPath/data """))
+
+    // wait until the task exited
+    waitForStatusUpdates(TASK_RUNNING, TASK_FINISHED)
+
+    // suspend the app
+    suspend(appId)
```

Comment:
Don't know if this is the most simple test case.
My initial idea would be to use a simple kill.
But your proposal solves the initial create and exists logic.